### DISCUSSION TITLE
How to stop generating empty Parquet files when there is no data.

### DISCUSSION THREAD

**Discussion by ShahBinoy** [slingdata-io/sling-cli#266] [General]
Hi @flarco ,

I have data being fetched from MySQL database using a custom query in Replication.

Lot of times there is no data coming back for a certain query, in such scenario, Sling creates empty parquet file, this throws off the collective scanning of data. 

Is there a way to not create parquet file if the row count returned is 0 ?

**Comment by flarco** [comment]
Ah ok, will fix that in next release, thanks.
